doc/introduction/chemistry.rst

```diff
@@ -154,7 +154,8 @@ where a quantum computer is used to prepare the trial wave function of a molecul
 the expectation value of the *electronic Hamiltonian*, while a classical optimizer is used to
 find its ground state.
 
-We can use :func:`~.vqe.cost` to automatically create the QNodes and define the cost function:
+We can use :func:`~.VQECost` to automatically create the required PennyLane QNodes and define 

```

Comment:
maybe should revert this, as this change is done in the `move-vqe` branch (also should now be class, not func)